Khan/wonder-blocks
# Add a11y documentation for Icons in storybook

## Summary:
Conduct research and add an accessibility section to the
Storybook documentation for wonder-blocks-icon.

I added a list of rules as well as two relevant demos.

Issue: https://khanacademy.atlassian.net/browse/WB-1361

## Test plan:
- Double check the documentation looks right at http://localhost:63824/?path=/docs/icon-accessibility--right-to-left-icons
- Click on the reference links at the bottom and confirm
  the link to the articles they're labeled as.
- Use a contrast checker to confirm that the blue icon passes
  and the light blue icon fails, as claimed.

<img width="889" alt="image" src="https://user-images.githubusercontent.com/13231763/224873071-fa51e716-1286-4eac-9194-631957d6aff5.png">
